src/test/java/com/dev/calculator/CalculatorTest.java

```diff
@@ -0,0 +1,49 @@
+package com.dev.calculator;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class CalculatorTest {
+    private Calculator calculator;
+
+    @BeforeEach
+    void setUp() {
+        calculator = new Calculator();
+    }
+
+    @Test
+    void addOperationOk() {
+        Assertions.assertEquals(2, calculator.calculate(1, 1, '+'));
+    }
+
+    @Test
+    void subtractOperationOk() {
+        Assertions.assertEquals(0, calculator.calculate(1, 1, '-'));
+    }
+
+    @Test
+    void multiplyOperationOk() {
+        Assertions.assertEquals(9, calculator.calculate(3, 3, '*'));
+    }
+
+    @Test
+    void divideOperationOk() {
+        Assertions.assertEquals(4, calculator.calculate(16, 4, '/'));
+    }
+
+    @Test
```

Comment:
```suggestion
    @Test(expected = RuntimeException.class)
```

Reply:
This solution is not working with Junit5